.github/issue_template.md

```diff
@@ -0,0 +1,13 @@
+## 1 What is the issue?
+*TODO*
```

Comment:
## 1 Current Behaviour

## 2 Expected Behaviour

Reply:
It's a good idea, but I don't think it's easy to explain *behaviour* in most cases. Also I think that filling out *Definition of done* would force people to think what needs to be done to get the expected behaviour. 

But I'm not sure - it's debatable. What do you guys think, @zhenik @dangernil @fredrikhgrelland (and ofc @pdmthorsrud)?

Reply:
>edit: is there a way to have several types of templates (im checking now)? So you can choose feature or bug. Because with bug I'd also like something like "Steps to Reproduce" for example

I just was about to say that. It's possible, and I can create a template for those as well.

Reply:
Yeah, that might be a nice way of doing it. But at the same time, Keep It Simple, Stupid. Let's see what the others think!

Reply:
It could be several templates for feature, question, bug, or enhancement. Check https://github.com/akka/alpakka/issues/new/choose , once you click create a new issue, there are will be 4 different options. 

PS. I agree with @pdmthorsrud that template should be simple

<img width="1588" alt="Screenshot 2020-10-16 at 08 53 20" src="https://user-images.githubusercontent.com/15572799/96222832-3f325280-0f8d-11eb-85c9-ab05b84960c8.png">

Reply:
Yeah, this is what both Claes and I were talking about I believe

Reply:
I really like this idea, great job!

Reply:
Sorry, I did not notice that. I would suggest making one general short template.
smth like 

```
### Short description
(What)
Please describe the feature you'd like to see in short.


### Details
(Why)
Explain your idea in more detail and possibly suggest different ways to achieve it. 
```

Reply:
Yeah, basically something along the lines of the first template proposed, then two more for bug and feature.
Something like
`bug`
```
## Current behaviour

## Expected behaviour

## Suggestion(s)/solution(s) [Optional]
```


`feature`
```
## Feature Description

## Why is it needed
```

But this is your turf Claes, go crazy :D

Reply:
I agree! Will update PR with new templates and structure!